Sources/Core/Client/Client.swift

```diff
@@ -156,7 +116,14 @@ public final class Client {
             Thread.callStackSymbols.forEach { ClientLogger.logger("", "", $0) }
         }
     }
-
+    
+    /// A subscription for websocket connection status.
+    /// - Parameter completion: a completion block (see `ClientCompletion`).
+    /// - Returns: a subscription.
+    public func connection(_ completion: @escaping ClientCompletion<WebSocket.Connection>) -> Subscription {
+        return rxConnection.bind(to: completion)
```

Comment:
`bindOnce` instead of `bind`?

Reply:
Nope. You are observing the connection state continuously.